support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -34,12 +34,20 @@ const testDetailsPromo = [
 		frequency: 'Monthly',
 		promoCode: 'PLAYWRIGHT_TEST_SPLUS_MONTHLY',
 		expectedPromoText: '£8/month for the first 3 months, then £10/month',
+		expectedCheckoutTotalText: '£10 £8/month',
```

Comment:
I haven't really found a way to work out how to check for <s>strikethrough<s>.

Reply:
I might see if we can add some useful content and check for that similar to Option 3 in [this article](https://www.webaxe.org/strikethrough-html-accessibility/).